### ISSUE TITLE
Change special character in config file

### ISSUE THREAD

**Issue by vality-ml** [oxen-io/session-pysogs#223]
TL;DR: Change the ≤ character in sogs.ini.sample l 77 to 'less than or equal to' or something similar. It will make a stock docker implementation of the standalone version work.

Longer version: Setting up the proxied version in a container is not really a good idea considering the use of systemd, but the standalone version should be fully compatible with docker k8s and docker compose if you ask me. However, sogs.ini.sample will error with something along the lines of: 

`Failed to load config: 'ascii' codec can't decode byte 0xe2 in position 2708: ordinal not in range(128) `

because of that special character in that file. Unless you make your dockerimage compatible with that character. After that line is changed/deleted, everything works just fine in a stock docker image. There is no reason why that exact character should be there.

Thank you for all your hard work making this very easy to setup!
